specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -1250,6 +1272,27 @@ model AcsMessageChannelEventError {
   channelMessage?: string;
 }
 
+/** Message Analysis Language Detection */
+model AcsMessageLanguageDetection {
+  /** The language of the message. */
+  language: string;
+
+  /** The confidence score of the language detected. */
+  confidenceScore: string;
+
+  /** The translation of the message. */
+  translation: string;
+}
+
+/** Message Analysis Sentiment */
+model AcsMessageSentiment {
+   /** The score of the analysis. */
+    score: number;
+
+    /** The description of the analysis. */
+    description: string;
+}
+
```

Comment:
Thanks for the TypeSpec! Would you please also compile the TypeSpec and add the generated SystemEvents.json to this PR: following the instructions [here](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/README.md#for-service-system-events-pr-approval)